podvm-mkosi/README.md

```diff
@@ -0,0 +1,46 @@
+# Podvm images with mkosi
+
+[mkosi](https://github.com/systemd/mkosi) builds a bootable OS image from scratch. This way, we have full control over every detail of the image, especially over the image format and boot process. On the long run, we will implement fully, bit-by-bit reproducible images with mkosi, and use measured boot and an immutable root FS to ensure the image integrity through remote attestation.
+
+## Prerequisites
+
+Currently, mksoi and other related tools are provided through a [Nix](https://nixos.org/) flake. Nix ensures all tools used in the build of the image are itself reproducible and pinned. mkosi requires a very recent systemd version, so using tools installed on the host is usually not possible.
+
+The recommended way to install Nix on Linux systems is the [Determinate Systems nix-installer](https://github.com/DeterminateSystems/nix-installer).
+
+
+## Building an image
+
+### Building PodVM binaries and place them in the binaries-tree
+
+```sh
```

Comment:
As far as I understand, this funkyness is manly in combination with `--platform` / cross-arch builds, right? Or isn't the given command working for you? I don't want to stick with this dockerfile for long and rather pull in the individual artifacts directly, but I thought it would be a good first step to give something familiar.

Reply:
I had to switch to the docker driver for buildx as default, otherwise itwould not find the local image built in the first step (the builder img)

Reply:
What exactly did you change? Somehow this works out of the box for me.

Reply:
```
$ docker buildx use default
```

Reply:
So would you suggest to push? Or should we set `--builder default` for the build?

Reply:
We could also just merge into a single Dockerfile.

Reply:
I think it's fine to add this line as a remark in the readme